engine/sc_main.cpp

```diff
@@ -282,12 +293,14 @@ int sim_t::main( const std::vector<std::string>& args )
     }
     catch ( const std::exception& )
     {
+      _print_version_info();
       fmt::print( "\n" );
       std::throw_with_nested( std::runtime_error( "Setup failure" ) );
     }
 
-    // print version info
-    print_version_info( *dbc );
+    // print version info if it hasn't been displayed already
+    if ( !version_displayed )
```

Comment:
Any reason this check is here twice?

Reply:
Forgot to remove it but I think the lambda is unnecessary anyway as code execution stop after throw_with_nested is called